### DISCUSSION TITLE
TheengsGateway on Raspberry Pi B+

### DISCUSSION THREAD

**Discussion by orzechszek** [theengs/gateway#56] [General]
Hello,

Does anybody install TheengsGateway on RPI B+ ? I have a strange issue. I've installed a gateway on a last RPi image. I run it with DEBUG mode with a systemd to check. It's working for about ~2,5h and it stops notifying without any additional info. Systemd process is normally active and working but my BT dongle is disconnected. To work again, sometimes process restart helps, but usually I need to restart whole PI. Had anybody such problem?

**Comment by koenvervloesem** [comment]
I haven't tried this on a Raspberry Pi B+ yet, but I did try Theengs Gateway on newer Raspberry Pi models with an external Bluetooth dongle and this worked. Do you see something suspicious in the output of `dmesg | grep hci0`? Or can you try with another Bluetooth dongle or another Raspberry Pi to see the difference?

**Reply by orzechszek** [reply]
Command doesn't show any interesting info.
```
bluetoothctl
Agent registered
[CHG] Controller XX:XX:XX:XX:XX:XX Pairable: yes
```
Unfortunately I don't have any other BT dongle or RPi :(
